frontend/src/types/index.ts

```diff
@@ -103,3 +98,11 @@ export interface DirectoryData {
     tooling: Tools;
   };
 }
+
+export interface DatabaseModel {
+  id?: string;
+  metaData: Result;
+  boardLink?: string;
+  createdAt?: number;
```

Comment:
Not sure if this is legacy or not, but I think we should store dates in an ISO format than milliseconds since epoch